prune.sh

```diff
@@ -9,6 +9,24 @@ grab_version() {
         xargs
 }
 
+grab_latest_version() {
+    conda search -c "${CHANNEL}" --platform "${platform}" "${PKG}" 2>/dev/null | \
+        grep "${CHANNEL}" | \
+        awk -F '  *' '{print $2}' | \
+        uniq | \
+        tail -n 1 | \
+        xargs
+}
+
+grab_builds() {
```

Comment:
nit: add a comment here (it returns the spec of the package of a specific version `$1`)

Reply:
> nit: add a comment here (it returns the spec of the package of a specific version `$1`)

yes, I modify the function name.